src/pages/unseoJang/Main/Aside.js

```diff
@@ -0,0 +1,229 @@
+import React, { Component } from 'react';
+import Recommend from './Recommend';
+
+class Aside extends Component {
+  constructor(props) {
+    super(props);
+    //this.state생성
+    this.state = {
+      recomendMember: [],
+    };
+  }
+  //목데이터
+  componentDidMount() {
+    fetch('http://localhost:3000/data/unseoJang/member.json', {
+      method: 'GET',
+    })
+      .then(res => {
+        return res.json();
+      })
```

Comment:
```suggestion
      .then(res => res.json())
```
arrow function에서 본문에 중괄호를 없이 쓰면 바로 return 한다는 의미이기 때문에 위와 같이 간결하게 표현할 수 있습니다!